WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.kt

```diff
@@ -176,23 +198,20 @@ class HelpActivity : AppCompatActivity() {
                 intent.putExtra(WordPress.SITE, selectedSite)
             }
 
-            val tagsList: ArrayList<String>? = if (SiteUtils.isBlockEditorDefaultForNewPost(selectedSite)) {
-                // construct a mutable list to add the Gutenberg related extra tag
-                val list = ArrayList<String>()
+            // construct a mutable list to add the related and extra tags
+            val tagsList = ArrayList<String>()
```

Comment:
I do agree it's a better approach, but I'm not sure how to pass `MutableList` with intent. We are using `ArrayList` because we can pass it with intent using `putStringArrayListExtra`.

Reply:
sounds good 👍